src/interactive-window/interactiveWindow.ts

```diff
@@ -371,66 +346,17 @@ export class InteractiveWindow implements IInteractiveWindowLoadable {
         this.updateSysInfoMessage(message, true, cellPromise);
     }
 
-    private async finishWithFailureMessage(error: Error, cellPromise: Promise<NotebookCell | undefined>) {
+    private async finishSysInfoWithFailureMessage(error: Error, cellPromise: Promise<NotebookCell>) {
```

Comment:
Split the finishSysInfo message into two, one for success and one for failures, was a little weird to have a union of IKernel and Error, and then look for `info` to determine which is which